src/check-registry.js

```diff
@@ -1,7 +1,7 @@
 const core = require('@actions/core');
 const axios = require('axios').default;
 const exec = require('@actions/exec');
-const { getRegistry } = require('./registry');
+const { getRegistry, TEMPLATE_STATUS_IN_VERIFICATION } = require('./registry');
```

Comment:
String 7: Can we make "https://github.com/" part configurable? I mean move value to some external configuration instead of hardcoding host in the lib?